packages/webcomponents/docs/helpers/ModularCheckoutLayout1Example.tsx

```diff
@@ -0,0 +1,478 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
Should this be using react? That doesn't seem right since the web component library should not have react in it. The examples should be simple HTML and CSS that we can also show the code samples for, while keeping it easy to understand